test.d

```diff
@@ -187,24 +184,26 @@ private:
 
     bool libclangExists (const ref Clang clang)
     {
-        auto libclangPath = Path.join(workingDirectory, clang.versionedLibclang);
-        return Path.exists(libclangPath);
+        auto libclangPath = buildNormalizedPath(workingDirectory, clang.versionedLibclang);
+        return exists(libclangPath);
     }
 
     void download (const ref Clang clang)
     {
         auto url = clang.baseUrl ~ clang.filename;
         auto dest = archivePath(clang.filename);
 
-        if (!Path.exists(dest))
-            Http.download(url, dest);
+        import std.net.curl;
```

Comment:
I would rather not bring in the dependency of curl. That's one reason I use Tango.